docs/evm/near-web3-provider.md

```diff
@@ -0,0 +1,182 @@
+
+NEAR Betanet Wallet:
+
+https://wallet.betanet.near.org
+
+At this point, the NEAR Wallet website's local storage contains a key for this account. When you want to interact with a frontend using your account, you'll typically have the frontend redirect to Wallet, requesting the creation a special function-call access key. After allowing the creation, the browser redirects back to the dApp frontend where local storage contains this new key.
+
+```js
+const { NearProvider } = require('near-web3-provider');
+// …
+const nearProvider = new NearProvider({
+  networkId: this.nearNetwork,
+  masterAccountId: 'yourname.betanet', // NEAR account with key that exists in local storage
+  keyStore: new nearAPI.keyStores.BrowserLocalStorageKeyStore(),
+});
+```
+
+### Contract abstraction, set the provider
+
+Web3 developers are able to use familiar libraries for contract abstraction. For instance, in the case of the Pet Shop example, a `Contract` object is instantiated using the NPM package [`web3-eth-contract`](https://www.npmjs.com/package/web3-eth-contract).
+
+Then a `Contract` object can use methods like [`send`](https://web3js.readthedocs.io/en/v1.2.11/web3-eth-contract.html#methods-mymethod-send) and [`call`](https://web3js.readthedocs.io/en/v1.2.11/web3-eth-contract.html#methods-mymethod-call). 
+
+```js
+const Contract = require('web3-eth-contract');
+const { NearProvider, utils } = require('near-web3-provider');
+const path = require('path');
+const credentialsPath = path.join(os.homedir(), '.near-credentials');
+
+const yourBetanetAccount = 'yourname.betanet'; // NEAR account with key that exists in local storage
+const nearProvider = new NearProvider({
+  networkId: this.nearNetwork,
+  masterAccountId: yourBetanetAccount, 
+  keyStore: new nearAPI.keyStores.UnencryptedFileSystemKeyStore(credentialsPath),
+});
+
+const myContractArtifact = require('path/to/build/contracts/MyContract.json');
+const networkId = nearProvider.version;
+const myContractAddress = myContractArtifact.networks[networkId].address;
+const myContract = new Contract(myContractArtifact.abi, myContractAddress, {
+  from: myContractAddress
+});
+myContract.setProvider(nearProvider);
+// view-only
+const viewResult = myContract.methods.myViewMethod().call();
+// signed
+const yourBetanetAccountAsEthAddress = utils.nearAccountToEvmAddress(yourBetanetAccount);
+const signedResult = myContract.methods.mutateStateMethod().send({from: yourBetanetAccountAsEthAddress});
+```
+
+This covers the basic usage of `near-web3-provider` on the frontend, allowing your Ethereum smart contracts to interact with a website.
+
+## Backend
+
+The [NEAR Pet Shop example](https://github.com/near-examples/near-pet-shop) demonstrates both of these. Please see the [Truffle page](/evm/truffle) for details on the backend approach.
+
+### NEAR CLI
+
+You can interact with an EVM smart contract with NEAR CLI, the command-line interface tool. With NodeJS version 12+, you may install this globally with:
```

Comment:
"with NEAR CLI" => "with the NEAR CLI"

Reply:
Hm, I think this one I'd like to pass on. Elsewhere we refer to NEAR CLI as a tool with a proper noun kind of feel.

Reply:
All right.